MAINTAINERS.md

```diff
@@ -22,3 +22,5 @@ The maintainers [watch](https://github.com/Particular/ServiceInsight/watchers) t
 - After a pull request is merged, the milestone representing the next version should be added to:
   - The issue(s) which the PR is addressing, *or*
   - The PR itself, if it is not addressing any pre-existing issue.
+- If merging using the Github UI, the pull requests must not be merged using the "Squash and Merge" option as the Squash and Merge option creates a ff mmerge.
```

Comment:
We've disabled the alternative merge style options for this repo so we see this:

![image](https://user-images.githubusercontent.com/677704/29812499-1a787a26-8ca7-11e7-846a-1fb2829c1876.png)

Do we still need to include this point?

Reply:
I think it's worth including as an explanation of the reason why it was disabled. I'll change the text to match an explaining tone.

Reply:
> I'll change the text to match an explaining tone.

@WilliamBZA is that change still pending?

Reply:
Nope, I forgot to do it!